server/utils/applications/getApplicationSubmissionData.ts

```diff
@@ -0,0 +1,29 @@
+import type { ApprovedPremisesApplication as Application, SubmitApplication } from '@approved-premises/api'
+import type { ReleaseTypeID } from '../../form-pages/apply/reasons-for-placement/basic-information/releaseType'
+import type { ApTypes } from '../../form-pages/apply/reasons-for-placement/type-of-ap/apType'
+
+import { retrieveOptionalQuestionResponseFromApplication, retrieveQuestionResponseFromApplication } from '../utils'
+
+export const getApplicationSubmissionData = (application: Application): SubmitApplication => {
```

Comment:
NBD but we could lose the 'get' here?

Reply:
Agreed - not having `get` prefixes seems to be the pattern we're adopting now